doc/manuals/admin/install.md

```diff
@@ -97,7 +97,7 @@ yum install contextBroker
 Sometimes the above commands fails due to yum cache. In that case, run
 `yum clean all` and try again.
 
-If you are upgrading using the RPM file, then first download the new package from the [FIWARE Files area](https://forge.fiware.org/frs/?group_id=7). Look for the "DATA-OrionContextBroker" entry.
+If you are upgrading using the RPM file, then first download the new package from the [FIWARE yum repository](https://nexus.lab.fiware.org/service/rest/repository/browse/el/7/x86_64/). Choose variant of package to install (nightly or release).
```

Comment:
again tell me how

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9